src/Files.App/Views/PaneHolderPage.xaml

```diff
@@ -70,84 +57,34 @@
 			NavParams="{x:Bind NavParamsLeft, Mode=OneWay}"
 			PaneHolder="{x:Bind}" />
 
-		<!--  Pane Right  -->
-		<Border
-			x:Name="PaneRightBorder"
-			Grid.Column="1"
-			x:Load="{x:Bind IsRightPaneVisible, Mode=OneWay}">
-			<shellpages:ModernShellPage
-				x:Name="PaneRight"
-				ContentChanged="Pane_ContentChanged"
-				Loaded="Pane_Loaded"
-				NavParams="{x:Bind NavParamsRight, Mode=OneWay}"
-				PaneHolder="{x:Bind}" />
-		</Border>
-
-		<!--  Pane Resizing Splitter  -->
+		<!--  Sizer  -->
 		<toolkit:GridSplitter
 			x:Name="PaneResizer"
 			Grid.Column="1"
-			Width="5"
-			MinWidth="5"
-			HorizontalAlignment="Left"
+			Width="4"
+			x:Load="{x:Bind IsRightPaneVisible, Mode=OneWay}"
 			Background="Transparent"
 			Canvas.ZIndex="150"
 			DoubleTapped="PaneResizer_OnDoubleTapped"
-			Foreground="Transparent"
-			GripperForeground="Transparent"
 			IsTabStop="False"
 			Loaded="PaneResizer_Loaded"
 			ManipulationCompleted="PaneResizer_ManipulationCompleted"
 			ManipulationStarted="PaneResizer_ManipulationStarted"
+			Opacity="0"
 			ResizeBehavior="BasedOnAlignment"
-			ResizeDirection="Auto"
-			Style="{StaticResource DefaultGridSplitterStyle}">
-			<toolkit:GridSplitter.RenderTransform>
-				<TranslateTransform X="0" />
-			</toolkit:GridSplitter.RenderTransform>
-		</toolkit:GridSplitter>
+			Style="{StaticResource DefaultGridSplitterStyle}" />
+
+		<!--  Pane Right  -->
+		<shellpages:ModernShellPage
+			x:Name="PaneRight"
+			Grid.Column="2"
+			x:Load="{x:Bind IsRightPaneVisible, Mode=OneWay}"
+			ContentChanged="Pane_ContentChanged"
+			Loaded="Pane_Loaded"
+			NavParams="{x:Bind NavParamsRight, Mode=OneWay}"
+			PaneHolder="{x:Bind}" />
 
 		<i:Interaction.Behaviors>
-			<icore:DataTriggerBehavior Binding="{x:Bind converters:MultiBooleanConverter.AndConvert(IsLeftPaneActive, IsRightPaneVisible), Mode=OneWay}" Value="True">
-				<icore:ChangePropertyAction
```

Comment:
Glad we can remove all these visual states! They were delicate to work with and had certain issues with theming.

Reply:
By the way those ones left can also be simplified by using Visual States.
You and I would say it is out of scope tho.
![image](https://github.com/files-community/Files/assets/62196528/cf173758-e0ca-46c6-89b7-6aa8c44a16f2)

Reply:
Correct

Reply:
We'll work on this more when we implement item templates.